src/transformation/utils/typescript/nodes.ts

```diff
@@ -23,3 +24,12 @@ export function isInDestructingAssignment(node: ts.Node): boolean {
             (ts.isBinaryExpression(node.parent) && ts.isArrayLiteralExpression(node.parent.left)))
     );
 }
+
+export function getNodeSymbol(context: TransformationContext, node: ts.Node): ts.Symbol | undefined {
```

Comment:
Maybe add a comment that using getSymbolAtLocation directly is preferred. Since this is a bit hacky after all.

And I would prefer `getSymbolForNode`.